Dolog rettelser (#44)
SpecialeBajs/Rapport: appendix/adj-implementation.tex

\section{Adjacency implementation}\label{app:adj-impl}

> The implementation of the adjacency matrix in LightGCN and Price-aware recommendation is described in this section.

Reviewer: ```suggestion
The implementations of the adjacency matrix in LightGCN and Price-aware recommendation are described in this section.
```

\subsection{LightGCN}

> The adjacency matrix is generated with SciPy that creates a dictionary of keys, where two keys indicate the position in the matrix, and the value is 1 if there is a connection.

Reviewer: ```suggestion
The adjacency matrix is implemented as a sparse matrix using SciPy that creates a dictionary of keys, where two keys indicate the position in the matrix, and the value is 1 if there is a connection.
```
